### PR TITLE
Sphereon support

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: address.config.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 module.exports = {
   whitelists: {
-    'nft.owner': ['0xa98A6eefbAE870b88a9C7A43f4b50066A01c93b6'],
-    'datatokens.address': []
+    'nft.owner': [
```
Comment (sksadjad) [inline address.config.js]: do we want to change the addresses in this file?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+services:
```
Comment (sksadjad) [inline docker-compose.yml:1]: I think this compose file better endup in the https://github.com/Sphereon/gx-deployment

File: src/@context/Asset.tsx [hunk 3]
```diff
@@ -187,7 +187,6 @@ function AssetProvider({
         const { verified, complianceApiVersion } = await verifyRawServiceSD(
           serviceSDContent
         )
-
```
Comment (sksadjad) [inline src/@context/Asset.tsx]: revert the unnecessary change

File: src/components/Asset/AssetContent/MetaFull.tsx [hunk 4]
```diff
@@ -5,10 +5,14 @@ import Publisher from '@shared/Publisher'
 import { useAsset } from '@context/Asset'
 import { getDummyWeb3 } from '@utils/web3'
 import { Asset, Datatoken, LoggerInstance } from '@oceanprotocol/lib'
+import { getPublisherNameOrOwner } from '@utils/ddo'
 
 export default function MetaFull({ ddo }: { ddo: Asset }): ReactElement {
   const [paymentCollector, setPaymentCollector] = useState<string>()
   const { isInPurgatory, assetState } = useAsset()
+  const complianceTypes = ddo?.metadata.additionalInformation.compliance
+  const isCompliant = complianceTypes?.length
```
Comment (sksadjad) [inline src/components/Asset/AssetContent/MetaFull.tsx:14]: suggest to check for the actual `gaia-x` in the list

File: src/components/Asset/AssetContent/MetaMain/MetaAsset.tsx [hunk 5]
```diff
@@ -6,23 +6,32 @@ import ExplorerLink from '@shared/ExplorerLink'
...
 }: {
   asset: AssetExtended
   isBlockscoutExplorer: boolean
+  verifiedServiceProviderName?: string
 }): ReactElement {
   const { isAssetNetwork } = useAsset()
   const { web3ProviderInfo } = useWeb3()
 
   const dataTokenSymbol = asset?.datatokens[0]?.symbol
+  const displayName =
```
Comment (sksadjad) [inline src/components/Asset/AssetContent/MetaMain/MetaAsset.tsx:24]: suggest to rename this variable to publisher/owner name

File: src/components/Search/utils.ts [hunk 6]
```diff
@@ -116,7 +117,13 @@ export function getSearchQuery(
     filters.push(getFilterTerm('services.type', accessType))
   serviceType !== undefined &&
     filters.push(getFilterTerm('metadata.type', serviceType))
-
+  complianceType !== undefined &&
```
Comment (sksadjad) [inline src/components/Search/utils.ts:120]: change this `complianceType !== undefined` to a more general check.

**Review by sksadjad — COMMENTED** [review]
@sanderPostma here is a few remarks about the changes

File: address.config.js (hunk 1 above)
Comment (sanderPostma) [reply address.config.js]: I've already reverted that, but the original does not work. I get "Could not retrieve asset" I will include a note in the PR with the address file I used.

File: docker-compose.yml (hunk 2 above)
Comment (sanderPostma) [reply docker-compose.yml:1]: Done in next commit

File: src/@context/Asset.tsx (hunk 3 above)
Comment (sanderPostma) [reply src/@context/Asset.tsx]: Outdated

File: src/components/Asset/AssetContent/MetaFull.tsx (hunk 4 above)
Comment (sanderPostma) [reply src/components/Asset/AssetContent/MetaFull.tsx:14]: Was done purposely. The flag is just to control the "(unverified)" part in the label. If another future compliance standard checker can verify the owner on another blockchain, that's fine too, it does not have to be Gaia-X perse.

File: src/components/Asset/AssetContent/MetaMain/MetaAsset.tsx (hunk 5 above)
Comment (sanderPostma) [reply src/components/Asset/AssetContent/MetaMain/MetaAsset.tsx:24]: Done, publisherNameOrOwner in next commit. (I still find it strange because these may be completely different entities.)
